language_dump.py

```diff
@@ -40,16 +44,36 @@
             if language_to_extract in data['strings'][string_key]:
                 if verbose:
                     print("Found existing translation for " + data['id'])
-                strings_by_object[data['id']][string_key] = data['strings'][string_key][language_to_extract]
+                strings_by_object[data['id']][string_key] = \
+                    data['strings'][string_key][language_to_extract]
             elif fallback_language in data['strings'][string_key]:
-                print("No existing translation for " + data['id'] + " yet, using English")
-                strings_by_object[data['id']][string_key] = data['strings'][string_key][fallback_language]
+                print("No existing translation for " + data['id'] + \
+                      " yet, using English")
```

Comment:
This might be a good opportunity to use f-strings.

Reply:
I didn't know about that, will do!